src/modules/main.js

```diff
@@ -0,0 +1,79 @@
+const inputBox = document.getElementById('input-box');
+const listContainer = document.getElementById('list-container');
+const STORAGE_KEY = 'list_data';
+
+// localStorage
+function saveData() {
+  localStorage.setItem(STORAGE_KEY, JSON.stringify(listContainer.innerHTML));
+}
```

Comment:
I must commend your innovative solution to the adding and removing of tasks. However, as requested by the previous reviewer, kindly save the tasks as an array of objects rather than the html element. This will allow you to be able to implement [this other requested change](https://github.com/LethaboMot/weppack2/pull/3/files#r1191672612).

Using an array of objects will work better since you'll be saving the properties of the tasks with which you can create the html element for each task. Remember that there's limit to the space available in local storage, so, the minimal the amount of data we save for each task, the better.